src/App.js

```diff
@@ -25,8 +34,36 @@ export default function App() {
     }
     getNewRecipe();
   }
-  function handleDelete(id) {
-    setRecipes(recipes.filter(recipe => recipe.id !== id));
+  async function handleDelete(obj) {
+    setRecipes(recipes.filter(recipe => recipe.id !== obj.id));
+    await fetch('/api/recipes', {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(obj._id),
+    });
+    alert('Recipe deleted');
```

Comment:
Maybe have a look at the library [react toastify](https://fkhadra.github.io/react-toastify/introduction/) 🙂